src/CarbonAware.WebApi/src/Models/EmissionsForecastDTO.cs

```diff
@@ -31,7 +31,7 @@ public record EmissionsForecastDTO : EmissionsForecastBaseDTO
     /// <summary>
     /// The forecasted data points transformed and filtered to reflect the specified time and window parameters.
     /// Points are ordered chronologically; Empty array if all data points were filtered out.
-    /// E.G. startTime and endTime outside the forecast period; windowSize greater than total duration of forecast data;
+    /// E.G. dataStartAt and dataEndAt times outside the forecast period; windowSize greater than total duration of forecast data;
```

Comment:
An additional thought: from conversations about another Batch API -- I think the EmissionsForecastDTO should include the original `RequestedAt` value that corresponds to the response object -- otherwise one cannot correlate the two.

Reply:
I think this makes sense and we'll work on these.